core/src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -425,80 +381,6 @@ object SampleWdl {
       """.stripMargin
     override lazy val rawInputs = Map.empty[String, String]
   }
-
-  object NestedScatterWdl extends SampleWdl {
```

Comment:
I should know and already forgot--nested scatters are definitely tested in Centaur?